<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1231</width>
      <height>621</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>GPIO</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>61</width>
        <height>551</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>GPIOA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIOB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIOC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIOD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIOE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GPIOF</string>
       </property>
      </item>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>51</width>
        <height>551</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Порт</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Пин</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>120</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Состояние</string>
      </property>
      <widget class="QCheckBox" name="GPIOPinON">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>94</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Включён</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>1081</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>211</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Направление порта</string>
      </property>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Вход</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>100</y>
        <width>181</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Функция порта</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Порт</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Основная функция</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Альтернативная функция</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Переопределенная функция</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>100</y>
        <width>141</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Тип порта</string>
      </property>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Аналоговый</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Цифровой</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>100</y>
        <width>241</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Подтяжка порта</string>
      </property>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Подтяжка к питанию</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>161</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Подтяжка к нулю</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Триггер Шмидта</string>
      </property>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Включен</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>200</y>
        <width>201</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Выходной драйвер</string>
      </property>
      <widget class="QRadioButton" name="radioButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>171</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Управляемый драйвер</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>171</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Открытый сток</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>200</y>
        <width>181</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Выходной фильтр</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Зарезервировано</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Медленный фронт</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Быстрый фронт</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Максимально быстрый фронт</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>200</y>
        <width>141</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Входной фильтр</string>
      </property>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>70</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Включен</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>190</y>
        <width>241</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Команды</string>
      </property>
      <widget class="QCheckBox" name="checkBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить команду включения</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>191</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить команды выключения</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action"/>
    <addaction name="FileOpen"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="FileOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
